docs/release_notes.rst

```diff
@@ -0,0 +1,71 @@
+Release notes
+=============
+
+2.2.0
+-----
+
+Bodhi 2.2.0 is a security and feature release, with a few bug fixes as well.
+
+
+Security
+^^^^^^^^
+
+This update addresses `CVE-2016-1000008 <https://github.com/fedora-infra/bodhi/pull/857>`_ by
+disallowing the re-use of solved captchas. Additionally, the captcha is
+`warped <https://github.com/fedora-infra/bodhi/commit/f0122855>`_ to make it more difficult to
+solve through automation.
+
+
+Features
+^^^^^^^^
+
+* Bodhi's ``approve_testing.py`` script will now comment on updates when they have reached a stable
+  karma threshold
+  (`5b0d1c7c <https://github.com/fedora-infra/bodhi/commit/5b0d1c7c>`_).
+* The web interface now displays a push to stable button when the karma reaches the right level when
+  autokarma is disabled
+  (`b65a6ccf <https://github.com/fedora-infra/bodhi/commit/b65a6ccf>`_).
```

Comment:
This commit fixes https://github.com/fedora-infra/bodhi/issues/772 and https://github.com/fedora-infra/bodhi/issues/796